### ISSUE TITLE
Implement Cross Validation Techniques for Evaluate the Recommenders

### ISSUE THREAD

**Issue by marcelcaraciolo** [muricoca/crab#56]
Check and implement several cross validation techniques for evaluate the recommenders.

https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/cross_validation.py

**Comment by marcelcaraciolo** [comment]
Implementing:

LeaveOneOut, LeavePOut, KFold, ShuffleSplit

**Comment by marcelcaraciolo** [comment]
Check the Pull #57

**Comment by marcelcaraciolo** [comment]
Merged.
